tests/Behat/Content/Node/ProjectContentContext.php

```diff
@@ -97,4 +112,50 @@ public function alterProjectFields(BeforeParseEntityFieldsScope $scope): void {
     ]);
   }
 
+  /**
+   * Creates media Remote video with the specified URLs.
+   *
+   * Usage example:
+   *
+   * Given the following remote videos:
+   *   | url   |
+   *   | url 1 |
+   *   |  ...  |
+   *
+   * @Given the following Remote video(s):
+   */
+  public function createMediaRemoteVideo(TableNode $table): void {
```

Comment:
Might be we should place it to build/modules/contrib/oe_media/tests/Behat/MediaContext.php? In this case we can reuse it in another tests and $this->cleanMedias() can be removed.

Reply:
don't think so, oembed is not part of oe_media

Reply:
Media Remote video is provided by oe_media module - build/modules/contrib/oe_media/config/install/media.type.remote_video.yml

Reply:
yes, your right

Reply:
I agree, let's move this into oe_media and reuse MediaContext.